### PR TITLE
Add copy updates for the procuration page

### PR DESCRIPTION
**Pull request by marineb**

### CODE REVIEW

File: src/AppBundle/Form/ProcurationElectionsType.php [hunk 1]
```diff
@@ -19,15 +19,19 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         $builder
             ->add('electionPresidentialFirstRound', CheckboxType::class, [
                 'required' => false,
+                'data' => false,
```
Comment (tgalopin) [inline src/AppBundle/Form/ProcurationElectionsType.php]: This shouln't be necessary. Did you have an issue?

Comment (marineb) [inline src/AppBundle/Form/ProcurationElectionsType.php]: @tgalopin there was a change request to set the checkboxes to `unchecked` by default. per the documentation, it seemed to be how to handle it. let me know your thoughts!

Comment (tgalopin) [inline src/AppBundle/Form/ProcurationElectionsType.php]: It's not here it should be modified then, but in the entity itself: https://github.com/EnMarche/en-marche.fr/blob/master/src/AppBundle/Entity/ProcurationRequest.php#L219

Comment (marineb) [inline src/AppBundle/Form/ProcurationElectionsType.php]: Oh I see. Thank you!

**Pull request merged**